airbyte-webapp/src/components/index.tsx

```diff
@@ -1,13 +1,11 @@
 export * from "./ArrayOfObjectsEditor";
-export * from "./DefaultLogoCatalog";
 export * from "./Label";
 export * from "./LabeledControl";
 export * from "./LabeledInput";
 export * from "./LabeledRadioButton";
 export * from "./LabeledSwitch";
-export * from "./Link";
+export * from "./common/Link";
 export * from "./LoadingPage";
-export * from "./MainPageWithScroll";
+export * from "./common/MainPageWithScroll";
```

Comment:
Should we maybe export everything from `src/common` inside `src/common/index.tsx`? Then we could just `export * from ./common` here

Reply:
This is a TODO. The goal is to eliminate these exports, but it would take a lot more file changes to the PR. I will follow up with these.